packages/phone-number-privacy/common/src/utils/responses.utils.ts

```diff
@@ -0,0 +1,35 @@
+import Logger from 'bunyan'
+import { Response } from 'express'
+import { FailureResponse, OdisRequest, OdisResponse, WarningMessage } from '..'
+
+// TODO: remove this once it is no longer being used by matchmaking
+export function respondWithError(
+  response: Response,
+  body: FailureResponse,
+  statusCode: number,
+  logger: Logger
+) {
+  const logObj = { error: body.error, statusCode }
```

Comment:
looks like `error` should be `err: body.error` in order to take advantage of bunyan's error formattig